resources/macos_user.rb

```diff
@@ -6,6 +6,7 @@
 property :autologin, [TrueClass]
 property :admin, [TrueClass]
 property :fullname, String
+property :groups, [Array, String]
```

Comment:
I have a small nit-pick with the property name here vs. what it does. If the property name is plural, that implies that it takes more than one thing. `groups admin` just doesn't seem correct from a semantic standpoint. Maybe having two distinct properties `group` and `groups` would be more clear? Thoughts?

Reply:
hmm...how about groups but always an Array whether it has 1 item or not? 

If someone were to ask "what groups are you in?", "I'm just in 1" is actually a fine answer too!